mok.c

```diff
@@ -166,6 +166,20 @@ struct mok_state_variable mok_state_variable_data[] = {
 		  MOK_VARIABLE_MEASURE,
 	 .pcr = 7,
 	},
+	{.name = L"MokTMLState",
+	 .name8 = "MokTMLState",
+	 .rtname = L"MokListTrustedRT",
+	 .rtname8 = "MokListTrustedRT",
```

Comment:
And here we've got "MokTMLState", but the RT variable is MokListTrustedRT".  Unless I've missed something major, "MokTML" vs "MokTMLState" is a bug.  In any case, I think the BS|NV variable needs to reflect the name of the RT variable; please make it "MokListTrusted".

Reply:
I'l change this to MokListTrusted